### PR TITLE
Playing the Code Reviewer

### PR DESCRIPTION
**Pull request by fdezcaminero**
Reviewing this styled html.

I get to act as a code reviewer in this code by my friend @IngDavidTM.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+     </div>
+   
+      </div>
+  
+          <div id="containerBox">
+         <div class="container-icon">
+             <i class="fa fa-3x fa-truck"></i>
+             <img src="https://www.iconsdb.com/icons/preview/orange/truck-2-xxl.png" alt="truck">
+          </div>
+         <div class="container-text">
```
Comment (fdezcaminero) [inline index.html:62]: Consider fixing your indentation.

Good indentation makes the code much easier to read and to understand. :1st_place_medal: 

Knowing which tag closes which will be very useful later on, both for others and yourself. :dove:

Depending on your OS and your text editor there will be different shortcuts to automatically indent your code. Try browsing the web for these shortcuts. :100:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  
+  
+  
+  
+  
+  
+  
+  
+    </div>
+  </body>
```
Comment (fdezcaminero) [inline index.html:161]: You should also consider eliminating unnecessary empty space. It can make the code confusing to read. :book:

**Review by fdezcaminero — COMMENTED** [review]
# Status: Required Change :recycle:



You are an extraordinary genius! :100:

So close!

## Highlights
- Amazing README file :heavy_check_mark:

## Required Changes ♻️

Please check the inline comments.

**Comment by fdezcaminero** [comment]
You only used divs. Try using semantic tags. They make the code easier to read and easier to inspect by search engines. :mag:

**Comment by fdezcaminero** [comment]
Please always try to check the linters before submitting your code for review. They save yourself a great deal of time. :watch:

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @fdezcaminero,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Polite and good use of emojis on your comments. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**